### PR TITLE
[webview] Limit popup message length. JB#51217

### PR DESCRIPTION
**Pull request by shanechko**

### CODE REVIEW

File: import/popups/PopupOpener.qml [hunk 1]
```diff
@@ -62,6 +62,11 @@ Timer {
             return false
         }
 
+        if (data.text.length > 300) {
```
Comment (rainemak) [inline import/popups/PopupOpener.qml:65]: I'm ok with this one. Let's just increase the length a bit. Maybe something like 1000? Any opinions?

File: import/popups/PopupOpener.qml [hunk 2]
```diff
@@ -62,6 +62,11 @@ Timer {
...
+            console.log("Message is too big to show and will be truncated.")
+            data.text  = data.text.substring(0,300) + " ..."
```
Comment (rainemak) [inline import/popups/PopupOpener.qml:67]: space missing after comma (0, 300)

File: import/popups/PopupOpener.qml (hunk 1 above)
Comment (rainemak) [reply import/popups/PopupOpener.qml:65]: + if (data.text &&...

Comment (shanechko) [reply import/popups/PopupOpener.qml]: done

File: import/popups/PopupOpener.qml (hunk 2 above)
Comment (shanechko) [reply import/popups/PopupOpener.qml]: ok

**Review by rainemak — APPROVED** [review]
LGTM

**Comment by rainemak** [comment]
Sorry. Didn't notice that this went to wrong branch. Will cherry-pick. The "next" branch is always the next coming version.

**Pull request merged**